lib/eventbrite/venue.rb

```diff
@@ -0,0 +1,35 @@
+class Eventbrite
+  class Venue
+    def initialize(details)
+      @details = details
+    end
+
+    def name
+      details["name"]
+    end
+
+    def address
+      [
+        street_and_city,
+        address_details["postal_code"]
+      ].reject(&:empty?).join(" ")
```

Comment:
Maybe overkill but how about a method wrapper for this array clean you're doing twice.